test/Bugs/misc_bugs.js

```diff
@@ -189,6 +189,28 @@ var tests = [

```

Comment:
nit: this would be a bit easier to read as `var ${'['.repeat(6631)}`

Reply:
Yup - that makes sense. Updated in the next commit